speed-dreams/source-2.2.3/src/modules/graphic/ssggraph/grboard.cpp

```diff
@@ -1550,6 +1555,43 @@ void cGrBoard::refreshBoard(tSituation *s, const cGrFrameInfo* frameInfo,
   if (trackMap) {
       trackMap->display(currCar, s, 0, 0, rightAnchor, TOP_ANCHOR);
   }
+
+}
+
+/// @brief Displays the texture stored in interventionTexture (which is loaded in grInitBoardCar, move this later)
```

Comment:
If this is something that needs to change maybe add TODO in front so it doesn't get lost